start.sh

```diff
@@ -7,26 +7,12 @@ if [ ! -f "./config/initializers/recaptcha.rb" ]; then
     echo -e "\e[94mRead More\e[0m: https://github.com/publiclab/mapknitter#installation-steps"
 fi
 
-bump_database(){
-	bundle exec rails db:schema:load || bundle exec rails db:migrate
-}
 
-bundle check || bundle install
-
-if bundle exec rails db:exists; then
-	>&2 echo "Database exists, only migrating it..."
-	bundle exec rails db:migrate
-else
-	>&2 echo "Database doesn't exist, creating and migrating it..."
-	bundle exec rails db:create
-	bump_database
-fi
-
-bundle exec rails assets:precompile
+DISABLE_DATABASE_ENVIRONMENT_CHECK=1 bundle exec rails assets:precompile
 
 if [ -f $pidfile ] ; then
 	>&2 echo 'Server PID file already exists. Removing it...';
 	rm $pidfile;
 fi
 
-forego start
+bundle exec rails s -p $PORT -b '0.0.0.0'
```

Comment:
We need to run rails using Forego otherwise ActionCable won't work. Check out https://github.com/publiclab/mapknitter/commit/cee7479cf4f0323a262b931a37cc72215ca48273